NEWS.md

```diff
@@ -96,7 +96,9 @@
 
 14. `setNumericRounding()` now invisibly returns the old rounding value instead of `NULL`, which is now consistent with similar behavior by `setwd()`, `options()`, etc. Thanks @MichaelChirico for the report and @joshhwuu for the fix.
 
-15. Extended support for non-standard evaluation (NSE) constructs within `[` and `melt()` functions. Now, constructs like `.`, `J`, `measure`, and `patterns` are exported, ensuring they are consistently available. Previously, only `.N`, `.I`, `.GRP`, `.GRPI`, `.SD`, and `:=` were exported. This update helps maintain consistency and functionality across your data.table usage, [#5604](https://github.com/Rdatatable/data.table/issues/5604) and [#5277](https://github.com/Rdatatable/data.table/issues/5277). Thanks @tdhock for suggestions and @Nj221102 for implementing this improvement.
+15. `.`, `J`, `measure`, and `patterns` are now exported to support non-standard evaluation (NSE) constructs within `[` and `melt()` functions. Previously, only `.N`, `.I`, `.GRP`, `.GRPI`, `.SD`, and `:=` were exported. This update ensures that these constructs are consistently available, maintaining consistency and functionality across your data.table usage. See [#5604](https://github.com/Rdatatable/data.table/issues/5604) and [#5277](https://github.com/Rdatatable/data.table/issues/5277). Thanks to @tdhock for the suggestions and @Nj221102 for implementing this improvement.
```

Comment:
I don't understand "This update ensures that these constructs are consistently available, maintaining consistency and functionality across your data.table usage." 
Isn't this relevant for package developers? Is it true that after merging this PR, a package function which uses `.` will no longer have to define it inside the function, to avoid a CRAN NOTE? That could be mentioned.

Reply:
Yes i agree, that merging this PR will enable package developers to not worry about CRAN NOTE while using these construct and i have changed the news item accordingly, I am sorry about taking this much reviewing for a news item but i think these reviews have made sure the news item is relevant to the users, thanks.